beam-sqlite/Database/Beam/Sqlite/Connection.hs

```diff
@@ -71,25 +70,24 @@ import           Data.ByteString.Builder (toLazyByteString)
 import qualified Data.ByteString.Char8 as BS
 import qualified Data.ByteString.Lazy.Char8 as BL
 import qualified Data.DList as D
-import           Data.Hashable (hash)
 import           Data.Int
 import           Data.Maybe (mapMaybe)
 import           Data.Proxy (Proxy(..))
 import           Data.Scientific (Scientific)
 import           Data.String (fromString)
 import qualified Data.Text as T
-import qualified Data.Text.Encoding as T (decodeUtf8)
 import qualified Data.Text.Lazy as TL
-import qualified Data.Text.Lazy.Encoding as TL (decodeUtf8)
 import           Data.Time ( LocalTime, UTCTime, Day
-                           , ZonedTime, utc, utcToLocalTime, getCurrentTime )
+                           , ZonedTime, utc, utcToLocalTime )
 import           Data.Typeable (cast)
 import           Data.Word
 import           GHC.TypeLits
 
 import           Network.URI
 
 import           Text.Read (readMaybe)
+import Data.IORef (newIORef, atomicModifyIORef')
+import GHC.IORef (atomicModifyIORef'_)
```

Comment:
This doesn't seem to fit the style of the existing import list.

Reply:
The comment `-- * Emulated @INSERT RETURNING@ support` should be updated as well (GitHub didn't let me include that line in the review).

Reply:
Thanks! I'm used to using `ormolu` so much that I forgot about the style